webapp/app/forms/steps/lotse/no_pauschbetrag.py

```diff
@@ -76,21 +76,17 @@ class StepNoPauschbetragPersonA(LotseFormSteuerlotseStep):
     preconditions = [HasDisabilityPersonAPrecondition,
                      HasNoPauschbetragOrFahrkostenpauschbetragClaimPersonAPrecondition]
 
-    def _pre_handle(self):
-        self._set_multiple_texts()
-        super()._pre_handle()
-
-    def _set_multiple_texts(self):
+    def render(self):
         num_of_users = get_number_of_users(self.render_info.stored_data)
-        self.title = ngettext('form.lotse.no_pauschbetrag.person_a.title',
+        step_title = ngettext('form.lotse.no_pauschbetrag.person_a.title',
                                                'form.lotse.no_pauschbetrag.person_a.title',
                                                num=num_of_users)
-
-    def render(self):
         props_dict = NoPauschbetragProps(
             step_header={
-                'title': str(self.title),
+                'title': str(step_title),
             },
+            showOverviewButton=bool(self.render_info.overview_url),
+            overviewUrl=self.render_info.overview_url,
```

Comment:
We could also only pass the overviewUrl and then just check for that in the component. However, I am fine with the more explicit way

Reply:
I think making it more explicit makes sense here because the parameter for link_overview is a boolean. Regardless of whether a URL was given, the parameter should be checked and not just the URL.